backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java

```diff
@@ -124,7 +147,169 @@ public String caseDefinition(Definition def) {
 
         return builder.toString();
     }
-    
+
+    public String caseLiteralExpression(LiteralExpression expression) {
+        Appender builder = this.newAppender();
+
+        if (expression instanceof LiteralInteger lit) {
+            builder.append(String.valueOf(lit.getValue()));
+        } else if (expression instanceof LiteralString lit) {
+            builder.append(lit.getValue());
+        } else if (expression instanceof LiteralRational lit) {
+            builder.append(String.valueOf(lit.getValue()));
+        } else if (expression instanceof LiteralBoolean lit) {
+            builder.append(String.valueOf(lit.isValue()));
+        } else if (expression instanceof LiteralInfinity lit) {
+            builder.append("*");
+        }
+
+        return builder.toString();
+    }
+
+    @Override
+    public String caseAttributeUsage(AttributeUsage attribute) {
+
+        Appender builder = this.newAppender();
+
+        this.appendUsagePrefix(builder, attribute);
+
+        builder.appendSpaceIfNeeded().append("attribute ");
+
+        this.appendUsageDeclaration(builder, attribute);
+
+        appendOwnedRelationshiptContent(builder, attribute);
+
+        return builder.toString();
+    }
+
+    private void appendFeatureSpecilizationPart(Appender builder, Feature feature) {
+        List<Redefinition> ownedRedefinition = feature.getOwnedRedefinition();
+        appendRedefinition(builder, ownedRedefinition, feature);
+
+        appendReferenceSubsetting(builder, feature.getOwnedReferenceSubsetting(), feature);
+
+        List<Subsetting> ownedSubsetting = new ArrayList<>(feature.getOwnedSubsetting());
+        ownedSubsetting.removeAll(ownedRedefinition);
+        ownedSubsetting.remove(feature.getOwnedReferenceSubsetting());
+
+        appendSubsettings(builder, ownedSubsetting, feature);
+
+        appendFeatureTyping(builder, feature.getOwnedTyping(), feature);
+        this.appendMultiplicityPart(builder, feature);
+    }
+
+    private void appendSubsettings(Appender builder, List<Subsetting> subSettings, Element element) {
+        if (!subSettings.isEmpty()) {
+            builder.appendSpaceIfNeeded().append(LabelConstants.SUBSETTING);
+            builder.appendSpaceIfNeeded().append(subSettings.stream().map(getSubsettedFeature())
+                    .filter(Objects::nonNull)
+                    .map(superFeature -> getDeresolvableName(superFeature, element))
+                    .collect(Collectors.joining(", ")));
+        }
+
+    }
+
+    // TODO : when the function Subsetting::getSubsettedFeature() is correctly implemented then remove this function and
```

Comment:
Please prefer a multi-line comment and remove the TODO

Reply:
done